utils/transaction-parser.ts

```diff
@@ -35,6 +37,38 @@ function parseZetaInstructionAccounts(
   return namedAccounts;
 }
 
+function enrichPlaceOrder(instructions: Instruction[], events: anchor.Event[]) {
+  if (events == undefined || events.length == 0) {
+    return;
+  }
+
+  // We can have multiple PlaceOrders in a single transaction, so keep track of how deep we've searched
```

Comment:
I don't think we have any SDK scenarios which will have multiple PlaceOrders in one transaction, so idk if I can test that. Seems to work fine for single PlaceOrders though 😅